config/source/multi/500-adapter.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: kafkasource-mt-adapter
+  namespace: knative-eventing
+  labels:
+    contrib.eventing.knative.dev/release: devel
+spec:
+  replicas: 1 # TODO: set to 0
```

Comment:
:grin:

Reply:
yup, WIP.

Reply:
This cause problems on the "pingsource" in combination with the operator,

see: 
* https://github.com/knative/operator/pull/381
* https://github.com/knative/eventing/issues/4627

let's have that in mind